packages/react/src/patterns/blocks/ContentGroupSimple/__stories__/ContentGroupSimple.stories.js

```diff
@@ -1,6 +1,7 @@
 import './index.scss';
 import { text, object, withKnobs, select } from '@storybook/addon-knobs';
 import ContentGroupSimple from '../ContentGroupSimple';
+import ContentGroupSimpleKnobs from './data/ContentGroupSimple.knobs';
```

Comment:
I think I see what you're trying to do, but we should probably link to it in the documentation at the least so people can easily see how the data needs to be structured.

Reply:
@photodow Yes for sure. I need to do this